packages/p2p/src/components/my-ads/my-ads-table.jsx

```diff
@@ -61,6 +61,19 @@ const MyAdsTable = () => {
                         />
                     </div>
                 )}
+                {general_store.banner_config.reduced_order_time && (
```

Comment:
Why are we checking this if it's always true?

Reply:
This banner needs to be removed in the next release. For a better handling, I am using a configuration, where I can control the display of banner. The config is not available from API nor we have a firebase where this can be configured. Hence, I am using it in store.